### PR TITLE
Explanation page prototype

### PR DESCRIPTION
**Pull request by Phoebe-Khokgawe**

### CODE REVIEW

File: portfolio/frontend/src/containers/SGonksPlatform/SGonksPlatform.js [hunk 1]
```diff
@@ -33,6 +37,7 @@ const SGonksPlatform = (props) => {
         <Route path="/mysgonks" exact component={MySGonks}></Route>
         <Route path="/competition" exact component={Competition}></Route>
         <Route path="/marketplace" exact component={Marketplace}></Route>
+        <Route path="/explanation" exact component={Explanation}></Route>
```
[the remark this answers is not in the document]
Comment (Phoebe-Khokgawe) [reply portfolio/frontend/src/containers/SGonksPlatform/SGonksPlatform.js:40]: yes

[the remark this answers is not in the document]
Comment (Phoebe-Khokgawe) [reply portfolio/frontend/src/containers/SGonksPlatform/SGonksPlatform.js:40]: I will fix that

[the remark this answers is not in the document]
Comment (texm) [reply portfolio/frontend/src/containers/SGonksPlatform/SGonksPlatform.js:40]: Why is this being removed?

Will review after this is resolved.

[the remark this answers is not in the document]
Comment (roonil-wazlib) [reply portfolio/frontend/src/containers/SGonksPlatform/SGonksPlatform.js:40]: That line was only there so the explanation page could be routed to easily for testing purposes, but it adds the page in somewhere it's not designed to be on the website.

[the remark this answers is not in the document]
Comment (Phoebe-Khokgawe) [reply portfolio/frontend/src/containers/SGonksPlatform/SGonksPlatform.js]: removed the code in this commit c4cf8fc0e55a00be6039b7e5d2d888c206845aa7

**Review by lin8231 — APPROVED** [review]
LGTM xd

**Review by leonnemets — APPROVED** [review]

**Pull request merged**